packages/rest/test/integration/rest.server.integration.ts

```diff
@@ -262,6 +269,119 @@ describe('RestServer (integration)', () => {
     expect(response.get('Access-Control-Allow-Credentials')).to.equal('true');
   });
 
+  it('exposes "GET /openapi.json" with openApiSpec.servers', async () => {
+    const server = await givenAServer({
+      rest: {
+        port: 0,
+        openApiSpec: {
+          servers: [{url: 'http://127.0.0.1:8080'}],
+        },
+      },
+    });
+    const greetSpec = {
+      responses: {
+        200: {
+          content: {'text/plain': {schema: {type: 'string'}}},
+          description: 'greeting of the day',
+        },
+      },
+    };
+    server.route(new Route('get', '/greet', greetSpec, function greet() {}));
+
+    const response = await createClientForHandler(server.requestHandler).get(
+      '/openapi.json',
+    );
+    expect(response.body).to.containDeep({
+      openapi: '3.0.0',
+      info: {
+        title: 'LoopBack Application',
+        version: '1.0.0',
+      },
+      servers: [{url: 'http://127.0.0.1:8080'}],
+      paths: {
```

Comment:
Please simplify this assertion to focus only on the part that's important (i.e. `servers` property) and leave out the rest of the API spec that's only adding noise to error messages and test source code.

```ts
expect(response.body.servers).to.deepEqual([
  {url: 'http://127.0.0.1:8080'},
]);
```